usecases.md

```diff
@@ -2,4 +2,471 @@
+
+- All requirements as listed for Use Case 1.C
+- Administrator, in consultation with trial participants, defines data sources, forecast run intervals/horizons, trial period, metrics, filters
+- Framework concatenates regularly posted forecast runs into a continuous forecast.
+- Framework accepts and allows for download of streaming data.
+- Framework generates intermediate reports during the trial.
+
+#### 1.G. Compare multiple forecast runs to measurements (stretch) {#uc1G}
+{: .anchor}
+
+**Use case narrative** : A framework user uploads a sequence of forecast runs at a regular interval, uploads corresponding data or selects reference data, specifies filters, specifies temporal averaging, and selects metrics. The framework calculates metrics, provides a visual display of forecast performance and an evaluation report for download.
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.A, for each forecast run.
+- Each forecast run must be identically specified.
+- Forecast runs are issued at a regular interval (e.g., every 6 hours).
+- User can upload multiple forecast runs. Each forecast run must be:
+  - A time series of uniform forecast intervals, duration, and value type (e.g. 5 minute interval, interval average forecasts through 24 hours from the start time)
+- Issued at a regular interval (e.g. every 6 hours).
+- User can define forecast evaluation intervals, horizons, and temporal averaging.
+
+
+
+### 2. Analyze forecasts {#analyzefx}
+{: .anchor}
+
+#### 2.A. Select subsets of forecasts and data {#uc2A}
+{: .anchor}
+
+**Use case narrative** : A framework user selects a subset of an uploaded forecast, and the corresponding data, based on one or more conditions, for detailed analysis.  The framework calculates metrics and provides a visual display of forecast performance.
+
+**Requirements** :
+
+- User can select a subset of points in a forecast or its corresponding data based on one or more conditions: forecast error (bias), forecast lead time, time of day, weather (irradiance, temperature, wind). Framework returns selected subsets from the forecast and corresponding data.
+- User can select among forecast performance metrics (3.A).
+- Framework applies data quality checks to uploaded data (4.E).
+- Framework provides metric values and visuals of forecast performance (3.A).
+- Framework provides a report for user download.
+
+
+
+#### 2.B. Identify events {#uc2B}
+{: .anchor}
+
+**Use case narrative** : The framework assists a user to identify events in a forecast or in data.
+
+**Requirements** :
+
+- User can upload a forecast.
+- User can upload data.
+- Framework applies data quality checks to uploaded data (4.E).
+- User can specify conditions (e.g., ramp rates) used to identify events occurring in a forecast or in data.
+- Framework provides visual display of identified events.
+- Framework provides a report for user download.
+- User can save event data, i.e., a time series of true/false values.
```

Comment:
@alorenzo175 are you ok with this requirement?

Reply:
yes, we can limit the number of event series if we need to